src/components/ActionBar/src/ActionBarButton.vue

```diff
@@ -295,25 +305,35 @@ export default {
 }
 
 .MainText {
-	display: flex;
 	gap: 8px;
-	align-items: center;
 	width: max-content;
-	max-width: 100%;
-	overflow: hidden;
-	line-height: 1;
-	white-space: nowrap;
-	text-overflow: ellipsis;
+}
+
+.iconButton .MainText {
+	display: flex;
+}
+
+@media screen and (max-width: 839px) {
```

Comment:
Mobile styles should always be the default base styles. Please move the styles within this media query into the root level and then refactor this query to be tablet/desktop specific overrides, i.e. `@media screen and (min-width: 840px) {}`.